src/routes/index.js

```diff
@@ -0,0 +1,10 @@
+import Search from '../pages/Search.svelte';
```

Comment:
According to Structure 
![image](https://github.com/user-attachments/assets/61ac7ba7-e910-4fc3-840b-b0123e7a6854)
 components should be in `routes` folder same as index.js for now I don't see a sense to separate them. @oleksandrstarov What do you think?

Reply:
Agree fixed it